release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -148,6 +148,104 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-storage"]
 === Storage
 
+[id="ocp-4-13-storage-user-managed-encryption"]
+==== User-managed encryption is generally available
+The user-managed encryption feature encrypts {product-title} node root volumes and the default storage class.
+
+When starting up managed {product-title} clusters, you can pass a custom Key Management Service (KMS) key to the installation program to encrypt the root volumes for the nodes in the cluster. The installation program also encrypts the default storage class with this key. This allows you to encrypt storage volumes with your selected key, instead of the platform’s default account key.
+
+This features supports the following storage types:
+
+* Amazon Web Services (AWS) Elastic Block storage (EBS) (for more information, see ???)
+
+* Microsoft Azure Disk storage (for more information, see ???)
+
+* Google Cloud Platform (GCP) persistent disk (PD) storage (for more information, see ???)
+
+[id="ocp-4-13-storage-csi-vol-health"]
+==== CSI volume health (Technology Preview)
+Container Storage Interface (CSI) volume health is currently enabled, and supported as a Technology Preview feature. However, {product-title} currently does not ship any CSI drivers that support this feature.
```

Comment:
Volume health tech preview has slipped to 4.14 or even later upstream.

Reply:
Removed.